tests/example_listings.py

```diff
@@ -34,6 +34,7 @@ def brief_response_data(essential_count=5, nice_to_have_count=5):
 
 def brief_data(essential_count=5, nice_to_have_count=5):
     return fixed_dictionaries({
+        'location': text(alphabet='abcdefghijkl'),
```

Comment:
We caught a unicode issue with the initial generator (https://github.com/alphagov/digitalmarketplace-api/pull/365/commits/857f8ff7a2b4c415fe369ca64c156a7c97b5a418) but then had to restrict the alphabet because it triggered some escaping error in SQLAlchemy with some of the strings.

We could've used a set of actual location choices, but since this doesn't go through schema validation there's no real reason to tying it to content values right now.